src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -29,6 +29,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
     </Content>
+    <Compile Include="CommandLine\CommitHashAttribute.cs" />
```

Comment:
At the very least consider placing this line next to other files from the same folder.

---
In reply to: [104981905](https://github.com/dotnet/roslyn/pull/17537#discussion_r104981905) [](ancestors = 104981905)